source/transfer_list.rst

```diff
@@ -657,5 +660,116 @@ such that the last ACPI table in this entry ends at offset
      - hdr_size
      - One or more ACPI tables.
 
+.. _optee_pagable_part_entry:
+
+OPTEE pagable part entry layout (XFERLIST_OPTEE_PAGABLE_PART)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This entry type holds the address of OPTEE pagable part.
+This address is used when OPTEED is the Secure Payload Dispatcher.
+
+.. _tab_optee_pagable_part:
+.. list-table:: OPTEE pagable part type layout
+   :widths: 2 2 2 8
+
+   * - Field
+     - Size (bytes)
+     - Offset (bytes)
+     - Description
+
+   * - tag_id
+     - 0x3
+     - 0x0
+     - The tag_id field must be set to **5**.
+
+   * - hdr_size
+     - 0x1
+     - 0x3
+     - |hdr_size_desc|
+
+   * - data_size
+     - 0x4
+     - 0x4
+     - The size of OPTEE pagable part in bytes.
+
+   * - pagable_part
+     - data_size
+     - hdr_size
+     - Pagable part address for OPTEED
+
```

Comment:
Do you not want to store the size of the pointed-to memory area anywhere? Is that not relevant for these?

Reply:
Not sure if I catch what you mean, but if for the size of 'pagable part address', it is stored in 'data_size'.

Reply:
Sorry, I think there may be a misunderstanding of how the `data_size` field works here? `data_size` always only contains the size of the TE itself (minus `hdr_size`). So if your TE just contains an 8-byte pointer to memory that's somewhere outside of the TL, then the `data_size` field for that TE must be set to 8. But you may still want to store the size of that pointed-to memory area somewhere (unless you really don't need that for your use case here?).

Reply:
Oh, I get what you mean now. No, OP-TEE just needs the base address, it doesn't need the size of the area that this address pointed to.